datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/GmsGraphQLEngine.java

```diff
@@ -1256,7 +1263,9 @@ private void configureTypeResolvers(final RuntimeWiring.Builder builder) {
                         .filter(graphType -> graphType instanceof EntityType)
                         .map(graphType -> (EntityType<?, ?>) graphType)
                         .collect(Collectors.toList())
-                )))
+                ))
+                .dataFetcher("exists", new EntityExistsResolver(entityService))
```

Comment:
I'm not sure we need to do this here. I think it should suffice to have resolvers registered on each individual entity type.